Documentation/hetzner-cicd.md

```diff
@@ -0,0 +1,43 @@
+# Hetzner Hosting GitLab CI/CD runners
+
+This document provide a summary of the Gitlab Runner infrastructure hosted
+in Hetzner hosts, both bare-metal and VMs.
+
+- [Hetzner Hosting GitLab CI/CD runners](#hetzner-hosting-gitlab-cicd-runners)
+  - [Node list](#node-list)
+    - [Hetzner EX130-R](#hetzner-ex130-r)
+    - [Hetzner AX41-NVMe](#hetzner-ax41-nvme)
+    - [Hetzner CAX21](#hetzner-cax21)
+
+
+## Node list
+
+### Hetzner EX130-R
+* Node type: bare-metal
+* IP: 49.12.169.71
+* Connection string:
+  ```bash
+  eval `ssh-agent` \
+  pass show mender/cicd/hetznercloud/gitlab-hetzner-ax41-runner-ssh_key-priv.pem | ssh-add - \
+  ssh root@49.12.169.71
+  ```
+
+### Hetzner AX41-NVMe
```

Comment:
this I couldn't find on the Hetzner site, but I guess that's just because of the availability of AX42...

Reply:
You're right! I guess so either